libvirt.go

```diff
@@ -1199,6 +1280,128 @@ func (l *Libvirt) Reset(dom string) error {
 	return nil
 }
 
+// BlockLimit contains a name and value pair for a Get/SetBlockIOTune limit.
+// The Name field is the name of the limit (to see a list of the limits that can
+// be applied, execute the 'blkdeviotune' command on a VM in virsh). The Value
+// field is the limit to apply.
+type BlockLimit struct {
+	Name  string
+	Value uint64
+}
+
+// SetBlockIOTune changes the per-device block I/O tunables within a guest.
+// Parameters are the name of the VM, the name of the disk device to which the
+// limits should be applied, and 1 or more BlockLimit structs containing the
+// actual limits.
+//
+// Possible limits which can be applied here include:
+// - total_bytes_sec
```

Comment:
missed a previous comment here? Can we make these constants?

Reply:
Wildo. I'm looking for the source from which they might eventually be generated.

Reply:
Done.